### PR TITLE
Branch3

### PR DESCRIPTION
**Pull request by rishikabera**

### CODE REVIEW

File: Answers.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+When git merges a branch to the checked out branch which is lot ahead of your checked out branch, we say git has fast-forwarded. For example a branch and a master branch is getting merged, the normal branch is way ahead of master branch at the time of merge and has only one commit before the merge. At the time of the merge, it 'fast-forwards' the master branch's pointer to the previous branch with merged commits.
+-----------------------------------------------------------------------------------------------------
+Answers 3.
+Branches in Git has proved to be beneficial because in a project, you can safely work in a branch without need to worry about affecting the man project. When everything in a branch is correct and conflicts resolved, you can then merge those code in the branch safely back to the main(or master) thus preventing any disastrous errors which might harm the overall project.
+--------------------------------------------------------------------------------------------------------------
+Answers 4.
+You can have as many repositiories you really want.
+In a project , usually we have the same  number of repositories as the count of the members in a project.
```
Comment (idhant96) [inline Answers.txt]: we cannot have the same number of repositories as a number of members... So how about  same number of branches as the number of members

File: Answers.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
```
Comment (idhant96) [inline Answers.txt:28]: This is a comment, so are you sure you want it in gitignore?

**Review by idhant96 — CHANGES_REQUESTED** [review]

File: Answers.txt (hunk 1 above)
Comment (rishikabera) [reply Answers.txt]: Usually one repository

Comment (rishikabera) [reply Answers.txt]: Made the required changes.

**Review by idhant96 — APPROVED** [review]

**Pull request merged**